ocp_install_env.sh

```diff
@@ -95,6 +95,11 @@ controlPlane:
 platform:
   baremetal:
     dnsVIP: ${DNS_VIP}
+    provisioningInterface: "ens3"
+    provisioningNetworkCIDR: "172.22.0.0/24"
+    provisioningIP: "172.22.0.3/24"
```

Comment:
All of these should have defaults in the installer PR, e.g see `pkg/types/baremetal/defaults/platform.go`